### PR TITLE
added slider component

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Review slider component

https://deploy-preview-16--vercelui.netlify.app/component-lib/slider.html

### CODE REVIEW

File: component-lib/slider.html [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+	<body>
+		<main class="main-container">
+			<header class="header">
+				<div class="flex-hz">
+					<button class="btn btn-icon btn-menu" onclick="openMobileSidebar()">
+						<i class="fa fa-bars fa-lg" aria-hidden="true"></i>
+					</button>
+					<a class="flex-hz link-no-style" href="../index.html">
+						<div class="logo-container">
+							<img class="img-round" src="./logo.png" />
```
Comment (gauravsinhaweb) [inline component-lib/slider.html]: use alt in img tag

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: component-lib/slider.html (hunk 1 above)
Comment (nikhiljugale007) [reply component-lib/slider.html]: okay. I will add.

**Comment by anasm11** [comment]
The square slider doesnt seem to work. Rest all looks good to merge.

**Comment by Naman-Saxena1** [comment]
> The square slider doesnt seem to work. Rest all looks good to merge.

+1 Code looks fine but slider is not working in Preview.

**Comment by nikhiljugale007** [comment]
> The square slider doesnt seem to work. Rest all looks good to merge.

It is working, just that it's taking a little time to load CSS I guess. 
On hover, the curser changes to the pointer. 
Can you check again?

File: component-lib/slider.html (hunk 1 above)
Comment (nikhiljugale007) [reply component-lib/slider.html]: Added alt tag in new commit

**Pull request merged**